Build/Scripts/BuildCommon.js

```diff
@@ -207,5 +213,61 @@ namespace('build', function() {
 
     });
 
+ 
+  task('gendocs', {
+    async: true
+    }, function() {
+
+    fs.copySync(atomicRoot + "Build/Docs/Readme.md", jsDocFolder + "Readme.md");
+    fs.copySync(atomicRoot + "Build/Docs/atomic-theme", jsDocFolder + "atomic-theme");
+
+    cmds = [
+      "cd " + jsDocFolder + " && echo {} > package.json", // newer versions of npm require package.json to be in the folder or else it searches up the heirarchy
```

Comment:
I dont know, but if there is a node available there is no reason it shouldn't. If not, it can removed from the windows build.